utam-core/src/main/java/utam/core/declarative/representation/PageObjectInterface.java

```diff
@@ -19,6 +19,8 @@
 
   Collection<MethodDeclaration> getDeclaredApi();
 
+  Collection<TypeProvider> getNestedInterfaces();
```

Comment:
i would add javadoc explaining this as it's more complex than others

Reply:
Fixed in next commit.